api/query/cache/index/index_filter_test.go

```diff
@@ -833,6 +833,65 @@ func TestBindExecute_TestLabel(t *testing.T) {
 	assert.Equal(t, expectedResult, srs[0])
 }
 
+func TestBindExecute_LabelWithWildcards(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	loader := NewMockReportLoader(ctrl)
+	idx, err := NewShardedWPTIndex(loader, testNumShards)
+	assert.Nil(t, err)
+
+	matchingTestName := "/a/b/c"
+	runs := mockTestRuns(loader, idx, []testRunData{
+		{
+			shared.TestRun{ID: 1},
+			&metrics.TestResultsReport{
+				Results: []*metrics.TestResults{
+					{
+						Test:   matchingTestName,
+						Status: "PASS",
+					},
+					{
+						Test:   "/d/e/f",
+						Status: "FAIL",
+					},
+				},
+			},
+		},
+	})
+	metadata := map[string][]string{
+		"/foo/bar/b.html": {"random"},
+		"/a/*":  {"interop1", "INTEROP2"},
```

Comment:
There's a possible edge case. Here's the scenario: If you modify `matchingTestName` to equal `/a` and this metadata line to `"/*":  {"interop1", "INTEROP2"},`, you will get zero results as-is.

The question I have: Is it even possible for the metadata to produce this scenario (if the appropriate meta.yml was modified)? If not, you can ignore this question.

*I don't think this would be a useful scenario to even have in the metadata (since it's just a root level wildcard) but I didn't want to assume.*

Reply:
Great question! It is a possible edge case, but it wouldn't make sense to have a root level wildcard, since it would label all WPT tests.  However, we should safeguard this case in the wpt-metadata repository.